### PR TITLE
Mary wants to run a 25 mile marathon

### PR DESCRIPTION
**Pull request by Anishluke92**

### CODE REVIEW

File: Mary_wants_to_run_a_25_mile_marathon.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  return true if total_distance == 25    
+  return false
+ end
+   
+
+    puts marathon_distance([1, 2, 3, 4])
+    puts marathon_distance([1, 9, 5, 8, 2])
+    puts marathon_distance([-6, 15, 4])
+    puts marathon_distance([])
```
Comment (danielpaul) [inline Mary_wants_to_run_a_25_mile_marathon.rb]: indented in too much

File: Mary_wants_to_run_a_25_mile_marathon.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+   Notes
+   Items in the array will always be integers.
+   Items in the array may be negative or positive, but since negative distance isn't possible, find a way to convert negative integers into positive integers.
+   Return false if the arguments are empty or not provided.
+=end
+   
+
+
+def marathon_distance(array)
+  return 0 if array.length < 0
```
Comment (danielpaul) [inline Mary_wants_to_run_a_25_mile_marathon.rb]: why would the method return 0 when all other cases the method returns true or false. the method should return numbers or a boolean. should not be returning two different data types.

File: Mary_wants_to_run_a_25_mile_marathon.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+  total_distance = 0
+  array.each do |elements|
+    total_distance += elements.abs()
+  end   
+
+  return true if total_distance == 25    
+  return false
```
Comment (danielpaul) [inline Mary_wants_to_run_a_25_mile_marathon.rb]:
```suggestion
  total_distance == 25    
```

Comment (danielpaul) [reply Mary_wants_to_run_a_25_mile_marathon.rb]: this is all you need. that already returns true or false.

**Review by danielpaul — CHANGES_REQUESTED** [review]

File: Mary_wants_to_run_a_25_mile_marathon.rb (hunk 3 above)
Comment (Anishluke92) [reply Mary_wants_to_run_a_25_mile_marathon.rb]: All the requested changes are fixed !

**Pull request merged**